CMakeLists.txt

```diff
@@ -94,13 +110,17 @@ add_subdirectory(tests)
 add_subdirectory(tools)
 
 if (UNIX)
-  add_subdirectory(3rdparty)
   add_subdirectory(debugger)
   add_subdirectory(docs/refman)
-  add_subdirectory(enclave)
-  add_subdirectory(enclave/core)
-  add_subdirectory(libc)
-  add_subdirectory(libcxx)
   add_subdirectory(pkgconfig)
   add_subdirectory(samples)
 endif ()
+
+# Add enclave directories
```

Comment:
I think it makes sense to declare these before the `if (UNIX)` logic. That way it reads: if we're building enclaves (which is always on *nix), include these folders. Then, if on *nix, additionally include these folders. (Later, if on Windows, include these...)

Reply:
Fixed.